### DISCUSSION TITLE
Intercom isn't found by ring-device-data-cli.js

### DISCUSSION THREAD

**Discussion by ujaehrig** [dgreif/ring#1171] [Q&A]
Hi,

I tried to access my Ring intercom via ring-client-api. But even though I can see it in the mobile app, when I run the ring-device-data-cli.js it returns with an empty JSON: 
```
{"locations":[{"name":"XXX","cameras":[],"chimes":[],"devices":[]}],"amazonKeyLocks":[]}
```

I've already removed the whole location and reconfigured it, but this didn't help either, although calling the cli when I had removed the location returned with an error, that the associated account doesn't have a location. 

Any other ideas, what I can try? 

Thank you, Ulf

**Comment by dgreif** [comment]
Thanks for the heads up! I'll get this fixed in the next release

**Reply by ujaehrig** [reply]
You think it's just a CLI problem? I had thought the lib might not detect it?

**Reply by dgreif** [reply]
The data-cli only logs specific fields. I forgot to add intercoms to it when I first added support for them. The latest version has been updated to log intercoms in the device discovery cli as well

**Reply by ujaehrig** [reply]
Great! Thank you!
